src/invoice2data/extract/templates/pl/pl.insert.subiekt-gt.yml

```diff
@@ -0,0 +1,43 @@
+# SPDX-License-Identifier: MIT
+keywords:
+  - 'Miejsce wystawienia:'
+  - 'Data wystawienia:'
+  - 'Sprzedawca:'
+  - 'Nabywca:'
+  - 'według stawki VAT'
+  - 'Razem do zapłaty:'
+  - 'Wystawił\(a\)'
+  - 'Odebrał\(a\)'
+  - 'Podpis osoby upoważnionej'
+fields:
+  issuer:
+    parser: regex
+    regex: Sprzedawca:.*\n(.*?)\s{3,}
+  vatin:
```

Comment:
I know there is no settlement yet on standard field names.
But is it ok, for you to rename this to `vat`?

Reply:
Personally don't have the knowledge if this can be replaced by vat.
Is it a personal or business vat number being parsed??

Currently not worthwhile blocking the pr for this.
When disered updates on the template can be made later on.